swift_browser_ui_frontend/src/common/lang.js

```diff
@@ -210,6 +211,8 @@ let default_translations = {
       upfail: "Failed uploading ",
       upnotsupported: "Uploading is not supported on your browser",
       copy: " Copy",
+      copied: "Copied!",
```

Comment:
I'd like to have more detailed message. For example:

```suggestion
      copied: "Share ID copied to clipboard",
```

Reply:
This sounds clearer to me as well. I changed the text.